<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetSecretWizard</class>
 <widget class="QWizard" name="SetSecretWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Snopf Secret</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <widget class="QWizardPage" name="page1">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="useExistingMnemonicRadio">
      <property name="text">
       <string>&amp;Type in existing mnemonic</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="createNewMnemonicRadio">
      <property name="text">
       <string>Create new &amp;mnemonic</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page2">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Make sure to keep these 24 words at a safe place!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="mnemonicEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page3">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Keep the button on Snopf pressed for five seconds.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>After the blinking LED stopped, click Next.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page4">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Press the button on Snopf to type in a test password:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="passwordEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="expectedPasswordLabel">
      <property name="text">
       <string>&lt;expected result&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
